docs/docs/platform/integrations.md

```diff
@@ -28,3 +30,9 @@ For email providers, you will be required to add the following data:
 ## Missing a provider?
 
 Novu is an open-source platform, meaning that if you are missing a particular provider you can create one using the guide specified [here](/community/create-provider).
+
+### Changelog
+
+:::warning
+In version 0.17.0 we introduced a breaking change in the Integrations API as the GET endpoints passed to bring the integrations filtered by environment identifier to bring the integrations filtered by organization.
+:::warning
```

Comment:
Is this a right place?

Reply:
I would agree, but I think a lot of people don't read the changelogs based on the support threads, so I was thinking the most the better. Trying to show the information by chance probability. 😄

Reply:
we should bring the changelog closer to the API docs